app/assets/src/components/common/DetailsSidebar/PipelineStepDetailsMode/pipeline_step_details_mode.scss

```diff
@@ -0,0 +1,49 @@
+@import "~styles/typography";
+@import "~styles/themes/typography";
+@import "~styles/themes/colors";
+
+.content {
+  padding: 30px;
+}
+
+.title {
+  font-size: $font-size-h4;
```

Comment:
you should use a mixin in themes/_typography.scss instead

Something like 

`@include font-header-xl` or font-header-l is probably what you want

Reply:
This goes for anything where you're setting font-size / line-height / letter-spacing / font-weight